akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamMarshalling.scala

```diff
@@ -38,7 +38,7 @@ object EventStreamMarshalling extends EventStreamMarshalling
 trait EventStreamMarshalling {
 
   implicit final def toResponseMarshaller(implicit ec: ExecutionContext): ToResponseMarshaller[Source[ServerSentEvent, Any]] =
-    Marshaller.withFixedCharset(MediaTypes.`text/event-stream`, HttpCharsets.`UTF-8`) { messages =>
+    Marshaller.withFixedContentType(ContentTypes.`text/event-stream`) { messages =>
```

Comment:
Ah, now I see. I missed the custom `ContentTypes` object. Well, that's unnecessary, because a `MediaTypes.WithFixedCharset` is implicitly converted to a `ContentType`.